modules/RTC/RTCUtils.js

```diff
@@ -888,10 +833,7 @@ class RTCUtils extends Listenable {
                         navigator.mediaDevices.getUserMedia.bind(
                             navigator.mediaDevices),
                             true);
-                this.enumerateDevices
-                    = wrapEnumerateDevices(
-                        navigator.mediaDevices.enumerateDevices.bind(
-                            navigator.mediaDevices));
+                this.enumerateDevices = rawEnumerateDevicesWithCallback;
```

Comment:
`rawEnumerateDevicesWithCallback` can be `undefined`.  Should we handle this somehow?

The old code will fail too if enumerateDevices is not defined. So this is not a regression for sure. I've just noticed it and decided to ask.